blocks/landing-page-hero/landing-page-hero.js

```diff
@@ -0,0 +1,194 @@
+                              <button class="video-overlay-close" aria-label="Close Button">
+                                <span class="vlt-icon-close-circle"></span>
+                              </button>
+                              <div class="video-overlay-video">
+                               <iframe allow="accelerometer; autoplay; encrypted-media; gyroscope;picture-in-picture" allowfullscreen="" class="video-overlay-iframe" title="Activate conversations everywhere with Vonage Business Communications" tabindex="0" id="video-iframe"></iframe>
+                              </div>
+                            </section>`;
+
+  const videoCloseButton = videoOverlay.querySelector('.video-overlay-close');
+  videoCloseButton.addEventListener('click', () => {
+    const videoContainer = (document.querySelector('.video-overlay'));
+    const videoPlayer = videoContainer.querySelector('iframe');
+    videoContainer.classList.remove('video-overlay-open');
+    videoPlayer.setAttribute('src', '');
+  });
+
+  return videoOverlay;
+}
+
+function assembleMediaContent(elements) {
+  const mediaContent = div({ class: 'media-content' });
+
+  // Iterate through the passed elements and append them to the media column
+  elements.forEach((element) => {
+    // If the element is a video reference set up the play icon and iframe video container
+    if (element.classList.contains('video')) {
+      mediaContent.appendChild(createVideoPlayButton(element));
+      mediaContent.appendChild(createVideoOverlay());
+    } else {
+      mediaContent.appendChild(element);
+    }
+  });
+  return mediaContent;
+}
+
+function buildOverlay(overlayRawContent) {
+  const overlayContent = div({ class: 'overlay' });
+
+  const overlayImage = overlayRawContent.querySelector(':scope picture');
+  const overlayLink = overlayRawContent.querySelector(':scope li > a');
+
+  overlayRawContent.querySelectorAll(':scope li > ul > li > code').forEach((styleRule) => {
+    overlayContent.style.cssText += styleRule.innerHTML;
+  });
+
+  overlayContent.appendChild(a({ class: 'overlay-link-wrapper', href: overlayLink.href }, overlayImage));
+
+  return overlayContent;
+}
+function decorateCtasContent(ctasContent, stylesList) {
+  stylesList.forEach((style) => {
+    if (style.includes('button')) {
+      ctasContent.classList.add(style);
+    }
+  });
+}
+
+export default function decorate(block) {
+  // If the markdown specifies a background property, need to bubble that up to the section wrapper
+  // So that the background spans the width of hte page
+  if (block.classList.contains('purple-gradient-background')) {
+    block.parentElement.parentElement.classList.add('purple-gradient-background');
+    block.classList.remove('purple-gradient-background');
+  }
+
+  const heroContainer = div({ class: block.classList });
+  // TODO: Potentially add logic so that you can control which side is the descriptor and which is the media via the order
```

Comment:
Yeah I had thought while initially implementing this block that it might be useful to allow for switching the category / media column (so the media could be on the left via a class specification for the block).  Don't see any place where that would actually be used in the clients site, so not worth implementing at this point.  I'll pull out the comment on my next commit.